### ISSUE TITLE
Add commands to allow and restrict roles

### ISSUE THREAD

**Issue by jopika** [jopika/Takane#17]
Create commands that would allow the modification of the whitelist/blacklist roles while the bot is still live.

```
!allowrole [role] - Allows the role to be modified by the !addrole and !removerole command
!restrictrole [role] - Restricts the role to be modified by the !addrole and !removerole command
```

**Comment by jopika** [comment]
Currently addressed in: 6008d0
Status; Waiting on Milestone Release

**Comment by jopika** [comment]
Completed and released: 479a7439faceec2317c5035b4dc78a123069b4dc
